src/public/scss/_wg.scss

```diff
@@ -2392,4 +2392,38 @@ $whitepurple: #eed5e3;
     padding: 20px;
   }
 
+  td.line-number {
+    text-align: right;
+    padding-right: 10px;
+    color: #6f777b;
+    font-family: monospace !important;
+    border-right: 1px solid #b5b5b5;
+    margin-right: 10px;
+  }
+
+  th.line-number + th {
+    padding-left: 20px;
+  }
+
+  td.line-number + td {
+    padding-left: 20px;
+  }
+
+  table {
+    position: relative;
+  }
+
+  th {
+    background-color: $white;
+    border-bottom: 1px solid #b5b5b5;
+    position: sticky;
+    top: 0;
+  }
+
+  div.alert-warning > details > table > thead > tr > th,
+  div.alert-warning > table > thead > tr > th {
+    background-color: $whitered;
+    border-bottom: 1px solid #b5b5b5;
+  }
```

Comment:
```
div.alert-warning > details > table > thead > tr > th,
div.alert-warning > table > thead > tr > th {
```

feels incredibly specific, is 

```
div.alert-warning th {
```

too broad?

Reply:
Probably not actually... Although I think following the review I did with Andy and Harry earlier I don't think you should end up in a position where you have a table in an alert box.  Let me remove this.